exec/java-exec/src/main/java/org/apache/drill/exec/vector/complex/MapVector.java

```diff
@@ -355,17 +353,19 @@ public void generateTestData(int values) { }
 
   @Override
   public void clear() {
-    valueCount = 0;
-    for (ValueVector v : getChildren()) {
+    for (final ValueVector v : getChildren()) {
```

Comment:
But clear() doesn't call vectors.clear(). I seem to recall a problem with adding that to clear() in the past. I'm worried this will introduce a regression, so I'd rather not take that change on now.